neo/public/javascripts/common/services.js

```diff
@@ -68,18 +222,18 @@ angular.module('cloudberry.common', [])
             asterixService.mapResult = result.value[1];
             asterixService.hashTagResult = result.value[2];
             break;
+          case "totalCount":
+            asterixService.totalCount = result.value[0][0].count;
+            break;
           case "error":
-            console.error(result);
+            console.error(result.toString());
```

Comment:
in JS print the object is better. since you can drill down to each nested field like in the debug mode.